guides/expression_language.md

```diff
@@ -163,14 +154,15 @@ let last_element = some_list[-1];
 // last_element is 42
 ```
 
-| Function | Parameter(s)                       | Description |
-| -------- | ------------                       | ----------- |
-| `get`    | position, counting from end if < 0 | gets a copy of the element at a certain position (`()` if the position is not valid) |
-| `len`    | _none_                             | returns the number of elements |
-| `filter` | predicate (usually a closure) | constructs a new array with all items that return `true` when called with the predicate function taking the following parameters:<ol><li>array item</li><li>_(optional)_ offset position</li></ol> |
-| `all`    | predicate (usually a closure) | returns `true` if all items return `true` when called with the predicate function taking the following parameters:<ol><li>array item</li><li>_(optional)_ offset position</li></ol>  |
+| Function | Parameter(s)                       | Description                                                                                                                                                                                        |
+| -------- | ---------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
Damn tables in md!!
Cannot they make something that doesn't require so many dashes XD